monkestation/code/modules/cassettes/cassette_approval.dm

```diff
@@ -131,8 +133,78 @@ GLOBAL_LIST_INIT(cassette_reviews, list())
 	message_admins("[submitter]'s tape has been approved by [user]")
 	logger.Log(LOG_CATEGORY_MUSIC, "[submitter]'s tape has been approved by [user]", list("approver" = user.name, "submitter" = submitter.name))
 	action_taken = TRUE
+	verdict = "APPROVED"
 
 /proc/fetch_review(id)
 	return GLOB.cassette_reviews[id]
 
 #undef ADMIN_OPEN_REVIEW
+
+// Handles UI to manage cassettes.
+/client/proc/review_cassettes() //Creates a verb for admins to open up the ui
+	set name = "Review Cassettes"
+	set desc = "Review this rounds cassettes."
+	set category = "Admin.Game"
+	var/datum/review_cassettes/tgui = new(usr)//create the datum
+	tgui.ui_interact(usr)//datum has a tgui component, here we open the window
```

Comment:
```suggestion
```
should prolly just get rid of this and call `ui_interact` in `/datum/review_cassettes/New` like in the other suggestion